### DISCUSSION TITLE
Failed to build finn-example

### DISCUSSION THREAD

**Discussion by ire06** [Xilinx/finn#818] [Q&A] [answered]
Hi

I am trying to install finn-examples on top of my PYNQ image(version 3.0.1). I'm using zcu111 board. However, during the installation I am getting the following error. "ERROR: Could not build wheels for finn-examples, which is required to install pyproject.toml-based projects" 
I followed the https://github.com/Xilinx/finn-examples/blob/main/README.md to set up the tools. Can someone please help.
Thank you

**Comment by mmrahorovic** [comment, marked as answer]
Hi @ire06,
We have tested and validated this flow for the ZCU104 board/image. Are you still experiencing the issues with the ZCU111 board?
If so, we'll have a deeper look into it.

Thanks for your question!
